Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -251,6 +265,22 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
                 return;
             }
 
+            if (this._variableType === WI.InlineSwatch.Type.Color) {
+                let colorVariable = this._delegate?.inlineSwatchColorVariableCompute(this, value.toString());
+
+                this._variableType = null;
+                this._type = WI.InlineSwatch.Type.Color;
+                this._value = colorVariable.value;
```

Comment:
what if there is no matching `colorVariable`?  should we have a `return` somewhere above in that case?